src/View.zig

```diff
@@ -0,0 +1,161 @@
+//! A View is effectively an "oversized" Window that can be written to and rendered in pieces.
+
+const std = @import("std");
+const mem = std.mem;
+
+const View = @This();
+
+const Screen = @import("Screen.zig");
+const Window = @import("Window.zig");
+const Unicode = @import("Unicode.zig");
+const Cell = @import("Cell.zig");
+
+/// View Allocator
+alloc: mem.Allocator,
+/// Underlying Screen
+screen: *Screen,
+/// Underlying Window
+win: *Window,
+
+pub const Config = struct {
+    max_width: usize = 10_000,
+    max_height: usize = 10_000,
+    x_pixel: usize = 0,
+    y_pixel: usize = 0,
+};
+pub fn init(alloc: mem.Allocator, unicode: *const Unicode, config: Config) !View {
+    const screen = try alloc.create(Screen);
```

Comment:
Does the `Screen` need to be allocated? I think we could get away storing the value rather than a pointer

Reply:
I think this depends entirely on how large the underlying Cell buffer gets, which is up to users since they'll no longer be bound by screen size. 

A potential solution is making the Allocator an Optional, putting it in the Config, and setting a sane limit for stack allocation before throwing a comptime error that forces the use of the Allocator.